app/views/layouts/admin_layout.html.erb

```diff
@@ -0,0 +1,89 @@
+<% content_for :head do %>
+  <%= stylesheet_link_tag 'admin_layout', media: 'all' %>
+  <%= javascript_include_tag "admin_layout" %>
+  <%= csrf_meta_tag %>
+<% end %>
+<%= render 'govuk_publishing_components/components/layout_for_admin',
+  product_name: "Collections Publisher",
+  environment: GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment,
+  browser_title: yield(:page_title) do %>
+
+  <%= render "govuk_publishing_components/components/skip_link" %>
+  <%= render "govuk_publishing_components/components/layout_header", {
+    product_name: "Publisher",
+    environment: GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment,
+    navigation_items: [
+      {
+        text: "Mainstream",
```

Comment:
I think this is supposed to be "Mainstream browse pages".

Reply:
Unfortunately, we need to use short names for now because using the full name breaks the navbar. We have a plan to add a secondary navigation bar.

Reply:
How much shorter does it have to be? Will "Mainstream browse" fit? 

Also, the name should be changed in both layouts for consistency.